### PR TITLE
Build on Xcode 6.3.1

### PR DESCRIPTION
**Pull request by pietbrauer**
I have seen some build errors with Travis and Circle CI on Xcode 6.3.1.

Error: `clang: error: invalid architecture 'arm' for deployment target '-mios-simulator-version-min=8.3'`

This is to experiment.

### CODE REVIEW

**Comment by pietbrauer** [comment]
@matthias-maennich Solved it. I don't know why I have to pass in the `destination` option here actually but know it builds. It also paves the way for #87

**Comment by dstnbrkr** [comment]
Ah, so it looks like #87 will not build on Xcode 6.2? I haven't had time to dig into why not if so, can you summarize?

**Comment by pietbrauer** [comment]
Travis default is still Xcode 6.1. #87 is is using swift 1.2 which is only available on Xcode 6.3. Building it for Xcode 6.2 won't build it Xcode 6.3.

**Comment by dstnbrkr** [comment]
Huh, so according to [this post](http://blog.travis-ci.com/2015-05-26-xcode-63-beta-general-availability/) that @nscoding found, 'beta-xcode6.3' is actually Xcode 6.3.1 on Travis. So this seems legit, let's merge.

**Comment by pietbrauer** [comment]
Oh sorry. That's what I meant actually.
